docs/community/translate.md

```diff
@@ -7,11 +7,39 @@ keywords:
     - Community
     - Subspace
     - Translate
+    - Localization
 ---
+import TranslationLeaderboard from '@site/src/components/TranslationLeaderboard/index.jsx';
 
 # Translation Guide
 
-This guide will show you how to provide translations for this documentation.
+Welcome to the Subspace Docs translation guide! This guide is here to help you contribute to our goal of making Subspace more accessible and inclusive by providing translations. Subspace Labs is driven by a vision of a decentralized and equitable future, and we believe that overcoming language barriers is crucial to achieving this vision.
```

Comment:
I wonder if that makes more sense to use Subspace Network or Subspace Labs here?

Reply:
Thanks for catching, got it adjusted to Network 🙏